supplementary_style_guide/glossary_terms_conventions/product_conventions/partner-and-hybrid-cloud-infrastructure.adoc

```diff
@@ -0,0 +1,762 @@
+
+In virtualization, "live migration" refers specifically to a virtual machine being moved from one host to another while remaining operational.
+
+*Use it*: With caution - for clarity, use the full term first before shortening to "migrate".
+
+[discrete]
+[[multipath]]
+==== image:images/caution.png[with caution] multipath (noun)
+*Description*: Device Mapper Multipath Input Output (DM-MPIO), also known as DM Multipathing or multipath, is a Linux utility that enables multiple input/output paths between server nodes and storage arrays to be configured into a single device path to provide redundancy and improve performance.
+
+*Use it*: With caution - this term may be unclear on its own.
+
+[discrete]
+[[n-way-replication]]
+==== image:images/yes.png[yes] n-way replication (noun)
+*Description*: Generically, refers to data being copied to multiple storage devices so that *n* copies are available in the storage cluster as a whole. This term is generally only used in Red Hat Gluster Storage.
+
+*Use it*: Yes
+
+
+[discrete]
+[[namespace-hci]]
+==== image:images/caution.png[with caution] namespace (noun)
+*Description*: An abstract group or context, generally used to identify and isolate resources from users, processes, and other resources that do not share the namespace. In the context of OpenShift, the term "project" is preferred over "namespace".
+
+*Use it*: With caution
+
+
+[discrete]
+[[network-encryption]]
+==== image:images/yes.png[yes] network encryption (noun)
+*Description*: The encoding of data that is in transit across a network, for the purpose of ensuring data security.
+
+*Use it*: Yes
+
+[discrete]
+[[network-time-configuration]]
+==== image:images/yes.png[yes] Network Time Configuration (noun)
+*Description*: Configuration for synchronizing time across all storage servers by using NTP (Network Time Protocol).
+
+*Use it*: Yes
+
+
+[discrete]
+[[object-storage]]
+==== image:images/yes.png[yes] object storage (noun)
+*Description*: A repository of data objects stored in an unstructured way, as distinct from block storage or file storage.
+
+*Use it*: Yes
+
+*See also*: xref:block-storage[block storage], xref:file-storage[file storage]
+
+[discrete]
+[[persistent-volume-hci]]
+==== image:images/yes.png[yes] persistent volume (noun)
+*Description*: A storage volume for data in an OpenShift cluster that remains available and usable after the pod that created it has been destroyed. A persistent volume provides persistent storage to OpenShift applications, as distinct from the ephemeral storage provided by default. This term is sometimes abbreviated as "PV", so it is important to clearly distinguish this term from "physical volume" in contexts where logical volume management is discussed, for example, when discussing storage for both Red Hat Enterprise Linux and OpenShift Container Platform.
+
+*Use it*: Yes
+
+[discrete]
+[[persistent-volume-claim-hci]]
+==== image:images/yes.png[yes] persistent volume claim (noun)
```

Comment:
We already do define this under the OCP glossary section. Same for "persistent volume" and probably some others.

We might need to think about how we display these terms. I think I get that the point of this section is to share cross-product storage terms, but if some of the terms are product-specific, we might want to leave the terms in those areas. And only worry about terms that really are used across products here. Thoughts?

Reply:
I think we may need an OpenShift person to weigh in on whether we can combine the definitions in this case; I think both are useful.